drake/addpath_drake.m

```diff
@@ -40,6 +40,10 @@
 addpath(fullfile(root,'systems','frames'));
 addpath(fullfile(root,'systems','visualizers'));
 addpath(fullfile(root,'systems','robotInterfaces'));
+addpath(fullfile(root,'systems','onlinePlanning'));
```

Comment:
sorry it took so long for me to get to this.  we should definitely get it in. 

i don't love this name, or location.  are all of these tools really so distinct?  we already have a trajectory library class (which hasn't been hit much, and perhaps should be replaced by yours) in systems/trajectories.  the funnel libraries could go there, too.